### ISSUE TITLE
loss=nan

### ISSUE THREAD

**Issue by Kurocuki** [niecongchong/Adjusted-HRNet-for-Semantic-Segmentation#1]
用该网络进行简单的建筑物提取, 但是loss=nan  loss设置的是`loss = keras.losses.BinaryCrossentropy()

        metrics = [keras.metrics.BinaryAccuracy()]
`
想问问为啥
